src/scripts/settings/userSettings.js

```diff
@@ -142,7 +142,7 @@ export class UserSettings {
         }
 
         // Enable it by default only for the platforms that play fMP4 for sure.
-        return toBoolean(this.get('preferFmp4HlsContainer', false), browser.safari || browser.firefox || browser.chrome || browser.edgeChromium);
+        return toBoolean(this.get('preferFmp4HlsContainer', false), browser.safari || browser.firefox || (browser.chrome && !browser.web0s && !browser.tizen) || browser.edgeChromium);
```

Comment:
fMP4 is enabled by default only for the following browsers. It is unchecked on others.
`(browser.safari || browser.firefox || browser.chrome || browser.edgeChromium)`

Perhaps you want to disable it completely to avoid confusing users?
https://github.com/jellyfin/jellyfin-web/blob/d60ddeb9edeb15d638d281923c06a573a74698e5/src/scripts/browserDeviceProfile.js#L747

Reply:
On my Tizen 4 TV, fMP4 requires the transcoding job to complete. But I have no idea if it works on modern models.

This checkbox is part (sort of) of the device profile settings, like `Enable DTS` and `Enable TrueHD`. So I agree with @GeorgeH005, we should leave it up to the user and add a warning.

Btw, Tizen isn't detected as Chrome, so this option wasn't turned on by default.